qualtran/bloqs/data_loading/select_swap_qrom.py

```diff
@@ -222,7 +225,7 @@ def batched_data(self) -> List[np.ndarray]:
         #
         # For data[N1][N2] with block sizes (k1, k2), you load batches of size `(k1, k2)` at once.
         # Thus, you load batch[N1/k1][N2/k2] where batch[i][j] = data[i*k1:(i + 1)*k1][j*k2:(j + 1)*k2]
-        batched_data = [np.empty(self.batched_data_shape) for _ in range(len(self.target_bitsizes))]
+        batched_data = [np.zeros(self.batched_data_shape, dtype=int) for _ in self.target_bitsizes]
```

Comment:
Why this change?

Reply:
`dtype=int` so we don't get floats by default. zeros instead of empty because the batched data size can be larger than actual data (to the closest power of 2)

Reply:
more on the empty -> zeros switch. Makes sense.